src/components/molecules/page-header/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { spacing, colors } from '../../../tokens'
+
+import { Heading1 } from '../../atoms/typography'
+import Breadcrumb from '../../atoms/breadcrumb'
+import Code from '../../atoms/code'
+
+import Description from './description'
+import Actions from './actions'
+
+const StyledPageHeader = styled.div`
+  margin-bottom: ${spacing.large};
+`
+
+const StyledLogo = styled.span`
+  width: 72px;
+  height: 72px;
+  display: inline-block;
+  float: left;
+  margin-right: ${spacing.small};
+  margin-top: ${spacing.xsmall};
+`
+
+const Type = styled.span`
+  text-transform: uppercase;
+  font-size: 12px;
+  color: #7c7c7c;
+  margin-right: ${spacing.small};
+`
+
+const ClientId = styled.span`
+  font-size: 13px;
+  color: #676767;
+`
+
+const PageHeader = props => {
+  let Top
+  if (props.actions) {
+    Top = <Actions>{props.actions}</Actions>
+  } else if (props.breadcrumb) {
+    Top = <Breadcrumb content={props.breadcrumb.content} link={props.breadcrumb.link} />
+  } else {
+    return null
+  }
+
+  return (
+    <StyledPageHeader>
+      {Top}
+
+      <div>
```

Comment:
In the case of the Client header (the complicated one), I think we could reorder some elements here to make it work with flexbox. I think that flexbox makes it easier to compose elements. We could organize the component like this:

```
<div> <<-- With flexbox
   <Logo /> 
  <div>
    <Heading>
    TYPE ... ID etc
  </div>
<div>
```

Reply:
wdty?